packages/cli/src/credentials/credentials.service.ee.ts

```diff
@@ -109,6 +109,11 @@ export class EnterpriseCredentialsService {
 		const { data: _, ...rest } = credential;
 
 		if (decryptedData) {
+			// We never want to expose the oauthTokenData to the frontend, but it
+			// expects it to check if the credential is already connected.
+			if (decryptedData?.oauthTokenData) {
+				decryptedData.oauthTokenData = true;
+			}
```

Comment:
Do we need to change/add a test for this `oauthTokenData`?

Reply:
Updated a test in `packages/cli/test/integration/credentials/credentials.api.ee.test.ts`